pywren/wren.py

```diff
@@ -71,17 +60,50 @@ def remote_executor(config=None, job_max_runtime=3600):
 
     AWS_REGION = config['account']['aws_region']
     SQS_QUEUE = config['standalone']['sqs_queue_name']
-    S3_BUCKET = config['s3']['bucket']
-    S3_PREFIX = config['s3']['pywren_prefix']
-    RUNTIME_S3_BUCKET = config['runtime']['s3_bucket']
-    RUNTIME_S3_KEY = config['runtime']['s3_key']
     invoker = invokers.SQSInvoker(AWS_REGION, SQS_QUEUE)
-    return Executor(AWS_REGION, S3_BUCKET, S3_PREFIX, invoker,
-                    RUNTIME_S3_BUCKET, RUNTIME_S3_KEY, job_max_runtime)
+
+    return Executor(invoker, config, job_max_runtime)
 
 standalone_executor = remote_executor
 
 
+def save_futures_to_string(futures):
```

Comment:
are we adding these functions in this PR ? Not sure it relates to the storage refactoring ? Can we move this out and discuss this separately ?